tests/config/nextflow.config

```diff
@@ -6,16 +6,31 @@ manifest {
 }
 
 params {
-    outdir = "output/"
+    outdir = "output"
     publish_dir_mode = "copy"
     singularity_pull_docker_container = false
-    test_data_base = './tests/data/qa'
-}
+    test_data_base = 'tests/data/qa'
+
+    // song/score setting 
+    api_token                  = null
+    song_url                   = null
+    score_url                  = null
+    song_url_download          = null
+    score_url_download         = null
+    song_url_upload            = null
+    score_url_upload           = null
+    transport_parallel         = null
+    transport_mem              = null
+    song_container             = null
+    song_container_version     = null
+    score_container            = null
+    score_container_version    = null
 
-process {
-    cpus = 2
-    memory = 3.GB
-    time = 2.h
+    // Max resource options
+    // Defaults only, expecting to be overwritten
+    max_memory                 = '3.GB'
+    max_cpus                   = 2
+    max_time                   = '1.h'
```

Comment:
Should we set it back to 2h? 
For reference: https://github.com/icgc-argo/workflow-roadmap/issues/290#issuecomment-1412167431

Reply:
These are all default settings, can be easily overwritten.